src/api/api.ts

```diff
@@ -1,57 +0,0 @@
-import axios, { AxiosRequestConfig } from "axios";
-import { onRequest, onRequestError, onRepoAccessRequest } from "./onRequest";
-
-export const BASE_URL = process.env.REACT_APP_BASE_URL;
-
-export const fetchData = axios.create({
-  baseURL: BASE_URL,
-  headers: {
-    "Content-type": "application/json",
-    Accept: "application/json",
-  },
-});
-
-export const postData = axios.create({
-  baseURL: BASE_URL,
-  headers: {
-    "Content-type": "text/plain;charset=utf-8",
-    Accept: "text/plain;charset=utf-8",
-  },
-});
-
-export const fetchImage = axios.create({
-  baseURL: BASE_URL,
-  headers: {
-    'Cache-Control': 'private'
-  }
-})
-
-export const getRepoAccess = axios.create({
-  baseURL: BASE_URL,
-  headers: {
-    "Content-type": "application/json",
-    Accept: "application/json",
-  },
-})
-
-export const postExternal = axios.create({
-  headers: {
-    "Content-type": "application/json",
-    Accept: "application/json",
-  },
-})
-
-fetchData.interceptors.request.use(
-  (config: AxiosRequestConfig) => onRequest(config),
-  onRequestError
-);
-
-postData.interceptors.request.use(
-  (config: AxiosRequestConfig) => onRequest(config),
-  onRequestError
-);
-
-getRepoAccess.interceptors.request.use(
-  (config: AxiosRequestConfig) => onRepoAccessRequest(config),
```

Comment:
Also needs TO BE used, to intercept the request , to make sure accessToken is set in the Authorization header, instead of authToken. To ensure connectivity to a private repository.

Reply:
These both in this file needs to be migrated as well as used in the app.

Reply:
I'm passing the accessToken header in the action that needs it:
https://github.com/input-output-hk/dapps-certification-web/blob/288fc2954503c70ce27ef919d1dabf7a0b7ba51e/src/store/slices/repositoryAccess.slice.ts#L37-L38

Reply:
But also, I'm adding a parameter for this too